src/IdServer/SimpleIdServer.IdServer/Services/Seeding/JsonSeedingService.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) SimpleIdServer. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using SimpleIdServer.IdServer.DTOs.Seeds;
+using SimpleIdServer.IdServer.Options;
+using SimpleIdServer.IdServer.Services.Seeding.Interfaces;
+using System.IO;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SimpleIdServer.IdServer.Services.Seeding
+{
+    /// <summary>
+    /// implements the methods to seed from an external resource.
+    /// </summary>
+    public class JsonSeedingService : ISeedingService
```

Comment:
To adhere to the convention in SimpleIdServer, could you add a folder named "Seeding" in "SimpleIdServer.IdServer" and move all the classes under "Services.Seeding" into this new folder? 
To reduce the number of files in the project and facilitate navigation, could you also move the "ISeedingService" interface into the "JsonSeedingService" class?

Reply:
Yes, no problem with that, I'll try to follow the project's conventions :)

Reply:
> To reduce the number of files in the project and facilitate navigation, could you also move the "ISeedingService" interface into the "JsonSeedingService" class

I think that it's better separated because in the future someone could create an implementation of "ISeedingService" for supporting another format like XML

Reply:
I created DTOs, I should move them too?